modules/connection.js

```diff
@@ -0,0 +1,22 @@
+var mysql=require('mysql');
+var connection=mysql.createConnection({
```

Comment:
put this in different config.json file, where you have to put the all the password and database name